lib/src/utils/crypto-utils.ts

```diff
@@ -43,8 +44,17 @@ export class CryptoUtils {
      *
      * @returns {string} code verifier.
      */
-    public static getCodeVerifier(): string {
-        return this.base64URLEncode(randombytes(32));
+    public static getCodeVerifier(env :string): string {
+        switch(env){
+            case "NODE":
```

Comment:
Can we replace these string literals with enums so that developers can refer to them easily? Check https://github.com/asgardeo/asgardeo-auth-js-sdk/blob/master/lib/src/constants/client-config.ts for reference